<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeView</class>
 <widget class="QWidget" name="HomeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget#timeTab{
	border: 1px solid #DCDCDC;
	border-radius: 15;
	background-color:  #DCDCDC;
	outline: 0px;
}
QListWidget#timeTab::item:hover{
	border: 1px solid #ffffff;
	border-radius: 14;
	background-color: #ffffff;
}
QListWidget#timeTab::item {
	border: 1px solid #DCDCDC;
	border-radius: 14;
}
QListWidget#timeTab::item:selected{
	background-color: #ffffff;
	border:1px solid #ffffff;
	show-decoration-selected: 0;
	border-radius: 14px;
	color: black;
}
QScrollArea,QWidget#recommend,QWidget#HomeView {
	border:none;
	background-color: transparent;
}
QWidget#fridayContents, QWidget#mondayContents, QWidget#tuesdayContents, QWidget#thursdayContents, QWidget#wednesdayContents, QWidget#saturdayContents, QWidget#sundayContents, QWidget#recentlyUpdatedContents,QWidget#friday, QWidget#monday, QWidget#tuesday, QWidget#thursday, QWidget#wednesday, QWidget#saturday, QWidget#sunday, QWidget#recentlyUpdated{
	border: 1px solid #F5F5F5;
	border-radius: 15px;
	background-color:  #F5F5F5;
}
QLabel#animaLabel1 {
	border-radius: 15px;
	background-color: #F5F5F5;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>    新番列表    </string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TabListWidget" name="timeTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="batchSize">
        <number>8</number>
       </property>
       <item>
        <property name="text">
         <string>最近更新</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周三</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周四</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周五</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周六</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>周日</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="timeAnimeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="recentlyUpdated">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="recentlyUpdatedContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="monday">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="mondayContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tuesday">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="tuesdayContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wednesday">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="wednesdayContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="thursday">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="thursdayContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="friday">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="fridayContents">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoScrollMargin">
          <number>2</number>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="saturday">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="saturdayContents"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sunday">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoListWidget" name="sundayContents"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="animaLabel1">
     <property name="text">
      <string>番剧推荐</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="recommend" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabListWidget</class>
   <extends>QListWidget</extends>
   <header>../util/widget/tabListWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>VideoListWidget</class>
   <extends>QListWidget</extends>
   <header>../util/widget/videoListWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>timeTab</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>timeAnimeView</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
